modules/checklists/checklists.php

```diff
@@ -955,12 +956,15 @@ public function save_global_checklist()
                 return;
             }
 
-            if (!wp_verify_nonce($_POST['_wpnonce'], 'ppch-global-checklists')) {
+            if (!wp_verify_nonce(sanitize_text_field($_POST['_wpnonce']), 'ppch-global-checklists')) {
```

Comment:
Nonce can be sanitized using `sanitize_key`

Reply:
updated